README.md

```diff
@@ -65,62 +65,115 @@ var result = sass.renderSync({
+Default: `[]`
 
-#### sourceComments
-`sourceComments` is a `Boolean` flag to determine what debug information is included in the output file.
+An array of paths that [libsass] can look in to attempt to resolve your `@import` declarations. When using `data`, it is recommended that you use this.
 
-#### sourceMap
-You must define this option as well as `outFile` in order to generate a source map. If it is set to `true`, the source map will be generated at the path provided in the `outFile` option. If set to a path (`String`), the source map will be generated at the provided path.
+### indentedSyntax
+Type: `Boolean`  
+Default: `false`
 
-#### sourceMapEmbed
-`sourceMapEmbed` is a `Boolean` flag to determine whether to embed `sourceMappingUrl` as data URI.
+`true` values enable [Sass Indented Syntax](http://sass-lang.com/documentation/file.INDENTED_SYNTAX.html) for parsing the data string or file.
 
-#### sourceMapContents
-`sourceMapContents` is a `Boolean` flag to determine whether to include `contents` in maps.
+### omitSourceMapUrl
+Type: `Boolean`  
+Default: `false`  
+**Special:** When using this, you should also specify `outFile` to avoid unexpected behavior.
 
-### The `render` Callback (>= v3.0.0)
+`true` values disable the inclusion of source map information in the output file.
+
+### outFile
+Type: `String | null`
+Default: `null`
+**Special:** Required when `sourceMap` is a truthy value
+
+Specify the intended location of the output file. Strongly recommended when outputting source maps so that they can properly refer back to their intended files.
+
+### outputStyle
+Type: `String`  
+Default: `nested`  
+Values: `nested`, `compressed`
+
+Determines the output format of the final CSS style. (`'expanded'` and `'compact'` are not currently supported by [libsass], but are planned in a future version.)
+
+### precision
+Type: `Integer`  
+Default: `5`
+
+Used to determine how many digits after the decimal will be allowed. For instance, if you had a decimal number of `1.23456789` and a precision of `5`, the result will be `1.23457` in the final CSS.
+
+### sourceComments
+Type: `Boolean`  
+Default: `false`
+
+`true` enables additional debugging information in the output file as CSS comments
+
+### sourceMap
+Type: `Boolean | String | undefined`  
+Default: `undefined`  
+**Special:** Setting the `sourceMap` option requires also setting the `outFile` option
+
+Enables the outputting of a source map during `render` and `renderSync`. When `sourceMap === true`, the value of `outFile` is used as the target output location for the source map. When `typeof sourceMap === "String"`, the value of `sourceMap` will be used as the writing location for the file.
+
+### sourceMapEmbed
+Type: `Boolean`  
+Default: `false`  
+
+`true` embeds the source map as a data URI
+
+### sourceMapContents
+Type: `Boolean`  
+Default: `false`  
+
+`true` includes the `contents` in the source map information
+
+## The `render` Callback (>= v3.0.0)
```

Comment:
How about removing `The` from the title, as you removed from the others as well?